tests/solver/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ set_target_properties(test_lid
 set(solver_test_srcs
     test_canonical.cpp
     test_gage.cpp
-    test_pollut.cpp
+    test_pollutants.cpp
```

Comment:
Why is rename necessary?

Reply:
Every thing else was fully spelt, so for consistency I updated the spelling from `test_pollut.cpp`